lib/manipulate_file.rb

```diff
@@ -17,20 +18,43 @@ def show_first_line
   end
 
   def mount_object
-    obj = { @name => { 'lines' => @file.length, 'players' => show_players_name } }
+    kills = show_players_kills
+    { @name => { 'lines' => @file.length, 'players' => show_players_name, 'kills' => kills[0],
+                 'total_kills' => kills[1] } }
```

Comment:
pequena melhoria, nível truque. Também mudei a formatação pra melhorar a leitura.

```suggestion
    kills, total_kills = show_players_kills
    {
      @name => {
        'lines' => @file.length,
        'players' => show_players_name,
        'kills' => kills,
        'total_kills' => total_kills
      }
    }
```